src/webgpu/api/validation/queue/writeBuffer.spec.ts

```diff
@@ -1,21 +1,12 @@
 export const description = `
 Tests writeBuffer validation.
 
-- buffer missing usage flag
-- bufferOffset {ok, unaligned, too large for buffer}
-- dataOffset {ok, too large for data}
-- buffer size {ok, too small for copy}
-- data size {ok, too small for copy}
-- size {aligned, unaligned}
-- size unspecified; default {ok, too large for buffer}
-
 Note: destroyed buffer is tested in destroyed/.
-
-TODO: implement usage flag validation.
-TODO: validate large write sizes that may overflow.
```

Comment:
This TODO is about `size` or `offset+size` overflowing a 64-bit integer, and I don't think we test it yet, so please leave the TODO (can add clarification) - or add the test cases in this PR.

Reply:
The maximum safe value of Number is 2^54 - 1, we cannot make `size` or `data size`  in bytes overflow a 64-bit integer, the value larger than Number.MAX_SAFE_INTEGER are not acceptable.

Reply:
That's the largest safe value, which means that every integer can be represented. After that, the next representable value after `x` is `x+2`, `x+4`, ..., `y`, `y+4`, `y+8`..., and so on. However you can still have values larger than MAX_SAFE_INTEGER that can be converted to uint64 (`unsigned long long`) by WebIDL, I believe.

The largest JS number (double) less than 2^64 appears to be `0xfffffffffffff800`, which should get converted to uint64, and then you could add an offset like `0x800` to it, and it would overflow to 0.